packages/onboarding/src/step-container-v2/components/StepContainerV2/StepContainerV2.tsx

```diff
@@ -50,15 +50,51 @@ export const StepContainerV2 = ( {
 		[ isMediumViewport, isLargeViewport ]
 	);
 
+	const topBarRef = useRef< HTMLDivElement >( null );
+	const [ topBarHeight, setTopBarHeight ] = useState( 0 );
+
+	const stickyBottomBarRef = useRef< HTMLDivElement >( null );
+	const [ stickyBottomBarHeight, setStickyBottomBarHeight ] = useState( 0 );
+
+	useLayoutEffect( () => {
+		const observer = new ResizeObserver( ( entries ) => {
+			entries.forEach( ( entry ) => {
+				const element = entry.target;
+
+				if ( element === topBarRef.current ) {
+					setTopBarHeight( entry.contentRect.height );
+				} else if ( element === stickyBottomBarRef.current ) {
+					setStickyBottomBarHeight( entry.contentRect.height );
+				}
+			} );
+		} );
+
+		if ( topBarRef.current ) {
+			observer.observe( topBarRef.current );
+		}
+		if ( stickyBottomBarRef.current ) {
+			observer.observe( stickyBottomBarRef.current );
+		}
```

Comment:
The `new ResizeObserver` callback doesn't get called immediately. So we should grab the dimensions here too.

```suggestion
		if ( topBarRef.current ) {
			observer.observe( topBarRef.current );
			setTopBarHeight( topBarRef.current.clientHeight );
		}
		if ( stickyBottomBarRef.current ) {
			observer.observe( stickyBottomBarRef.current );
			setStickyBottomBarHeight( stickyBottomBarRef.current.clientHeight );
		}
```